src/components/Swapping.tsx

```diff
@@ -457,6 +462,13 @@ const Swapping = ({ route, updateRoute, onSwapDone }: SwappingProps) => {
       )
     }
 
+    const chainSwitchRequired = steps.some(
+      (step) => step.execution?.status === 'CHAIN_SWITCH_REQUIRED',
```

Comment:
Should we maybe switch to enums for the status types so we don't have the ugly strings lying around here? Would also be nicer for sdk users if they have proper autocompletion I would say